Sources/Modules/CollaborationsModule.swift

```diff
@@ -153,16 +154,17 @@ public class CollaborationsModule {
                 "id": itemId,
                 "type": itemType
             ],
-            "role": role.description
+            "role": role.description,
+            "accessible_by": [
+                "login": login,
+                "type": "user"
+            ]
```

Comment:
```suggestion
            "accessible_by": [
                "login": login,
                "type": "user"
            ],
            "role": role.description
```

Reply:
[nit] What do you think about putting the JSON values in this request in the same order as the response?

(I'm assuming that the test below is saying that `accessible_by` comes before `role`, right?